pkg/steps/release/release_images.go

```diff
@@ -132,7 +132,10 @@ func (s *releaseImagesTagStep) Inputs(ctx context.Context, dry bool) (api.InputD
 }
 
 func sourceName(config api.ReleaseTagConfiguration) string {
```

Comment:
I would expect this to come with a unit test

Reply:
Debugging is done.
Removed the change.